src/common/Common.h

```diff
@@ -993,6 +993,16 @@ namespace Sapphire::Common
     CircularAoEPlaced = 7
   };
 
+  enum City : uint8_t
```

Comment:
use this definition instead: https://github.com/SapphireServer/Sapphire/blob/master/src/common/CommonGen.h#L559-L568